src/components/Review.js

```diff
@@ -24,7 +24,7 @@ export default function Review({
       </EditButton>
       <Title>{title}</Title>
       <Tags>
-        <Tag>{category}</Tag>
+        <Tag data-cy="category">{category}</Tag>
```

Comment:
Why do you need a data-cy for testing your category, but not for subcategory and rating? Could you maybe get the "category" by the label (like you did with subcategory and rating in your cypress test)?